checker/bin-devel/build.sh

```diff
@@ -81,6 +81,6 @@ fi
 (./gradlew help || sleep 10) > /dev/null 2>&1
 
 echo "running \"./gradlew assemble\" for checker-framework"
-./gradlew assemble --console=plain --warning-mode=all -s --no-daemon -Dorg.gradle.internal.http.socketTimeout=60000 -Dorg.gradle.internal.http.connectionTimeout=60000
+./gradlew assemble --console=plain --warning-mode=all -s --daemon -Dorg.gradle.internal.http.socketTimeout=60000 -Dorg.gradle.internal.http.connectionTimeout=60000
```

Comment:
Since running the daemon is the default, it is better to remove `--no-daemon` rather than replacing it with `--daemon`.